_includes/header.html

```diff
@@ -9,19 +9,49 @@
     </div>
     <div id="navbar-menu" class="navbar-menu is-size-5">
       <div class="navbar-start">
-        <a class="navbar-item {% if page.url == '/' %}is-active{% endif %}" href="{{ site.baseurl }}/">Accueil</a>
-        <a class="navbar-item {% if page.url == '/principles' %}is-active{% endif %}" href="{{ site.baseurl }}/principles">L'appel et les 9 principes</a>
+        {% assign homeUrl = '/' | prepend: page.lang | prepend: '/' %}
+        <a class="navbar-item {% if page.url == homeUrl %}is-active{% endif %}"
+          href="{{ homeUrl | relative_url }}">{{ site.data.translations.menu.home[page.lang] }}</a>
```

Comment:
Why `relative_url`?

Reply:
To keep everything working if we set a value to baseUrl.